src/main/java/gift/entity/Product.java

```diff
@@ -18,13 +18,12 @@ public class Product {
     @Id
     private Long id;
     @NotNull
-    @Size(max = 15, message = "상품 이름은 공백을 포함하여 최대 15자까지 입력할 수 있습니다.")
-    @Pattern(regexp = "^[a-zA-Z0-9가-힣 ()\\[\\]+\\-&/_]*$", message = "(),[],+,-,&,/,_를 제외한 특수 문자 사용은 불가능합니다.")
-    @Pattern(regexp = "^(?!.*카카오).*$", message = "카카오가 포함된 문구는 담당 MD와 협의한 경우에만 사용할 수 있습니다.")
     private String name;
     @NotNull
-    private int price;
+    private Integer price;
     private String img;
+    @OneToMany(mappedBy = "product")
+    private List<Wish> wishes;
```

Comment:
NPE를 방지하기 위해 생성까지 해주는 편입니다.
```suggestion
    private List<Wish> wishes = new ArrayList();
```